tests/fixtures/netexidf2ntfs/output/comment_links.txt

```diff
@@ -1,3 +1,3 @@
 object_id,object_type,comment_id
-prefix:stif:VehicleJourney:local-26-C00114-1:LOC,trip,prefix:STIF:Notice:C00164_7:LOC
```

Comment:
Nothing changed in this file, why the difference then?

Reply:
nothing, I just execute the binary on the fixtures and I know there is a BtreeSet that reorder lines. I'll change it.